app/views/polls/_lecturer_table.html.erb

```diff
@@ -13,14 +13,18 @@
   <% polls.sort_by{|poll| poll.created_at}.each do |poll| %>
     <tr>
       <td><%= poll.title %></td>
-      <td><%= poll.is_active %></td>
+      <td><%= poll.status %></td>
       <td class="text-right">
         <% if @lecture.allow_interactions?%>
-          <%= link_to 'Edit', edit_course_lecture_poll_path(poll.lecture.course, poll.lecture, poll), {class: "btn btn-primary"} %>
-          <% text = poll.is_active ? 'Stop' : 'Start' %>
-          <%= link_to text, stop_start_course_lecture_poll_path(poll.lecture.course, poll.lecture, poll), {class: "btn btn-primary"} %>
+          <%= link_to 'Edit', edit_course_lecture_poll_path(poll.lecture.course, poll.lecture, poll), {class: "btn btn-secondary"} %>
+          <% if poll.status == "created" %>
+            <%= link_to 'Start', stop_start_course_lecture_poll_path(poll.lecture.course, poll.lecture, poll), {class: "btn btn-primary"} %>
+          <% elsif poll.status == "running"%>
+            <%= link_to 'View', course_lecture_poll_path(poll.lecture.course, poll.lecture, poll), {class: "btn btn-primary"} %>
+          <% else %>
+            <%= link_to 'View', course_lecture_poll_path(poll.lecture.course, poll.lecture, poll), {class: "btn btn-primary"} %>
```

Comment:
I do not see any difference between these last two cases. Please convert them into one case :slightly_smiling_face: